src/Elliptic/Executables/Elasticity/SolveElasticity.hpp

```diff
@@ -6,6 +6,9 @@
 #include <cstddef>
 
 #include "DataStructures/DataBox/PrefixHelpers.hpp"
+#include "Domain/Creators/Factory1D.hpp"
+#include "Domain/Creators/Factory2D.hpp"
+#include "Domain/Creators/Factory3D.hpp"
 #include "Domain/Creators/RegisterDerivedWithCharm.hpp"
```

Comment:
Is the registration actually necessary for the domain creators / do you why it is? Since the domain creators are not PUPable it seems to me that Charm doesn't need to know about the subclasses.

Reply:
This actually registers the maps, not the creators.  I agree that the name is somewhat misleading.

Reply:
Oh I see, fine then